aws-ec2-knownhosts.cabal

```diff
@@ -22,17 +22,17 @@ library
                  , AWS.PubKeys
                  , AWS.Types
 
-  build-depends: base        >= 4.7   && < 4.13
-               , aeson       >= 1.0   && < 1.5
+  build-depends: base        >= 4.7   && < 4.16
+               , aeson       >= 1.0   && < 2.1
                , async       >= 2.0   && < 2.3
-               , attoparsec  >= 0.12  && < 0.14
+               , attoparsec  >= 0.12  && < 0.15
                , bytestring  >= 0.10  && < 0.11
                , directory   >= 1.2   && < 1.4
                , filepath    >= 1.4   && < 1.5
                , io-streams  >= 1.3.4 && < 1.6
                , process     >= 1.4   && < 1.7
                , text        >= 1.1   && < 1.3
-               , text-format >= 0.3   && < 0.4
+               , text-format >= 0.3.3   && < 0.4
```

Comment:
I'm wondering what to do about these version bounds.  The lower version bounds are possibly no longer correct.  I haven't tested this library with anything older than LTS-19.

Should I just remove the lower version bounds, since we don't really have any automated way to figure out correct version bounds?

Or should I bump the lower version bounds to match the package versions in LTS-19?  This would be technically correct, although likely too strict.

Reply:
This is an excellent question... I'd rather keep it open for now and if somebody complains (they won't), then we tighten it up 👍